repos/system_upgrade/el7toel8/actors/checkpcidevices/actor.py

```diff
@@ -0,0 +1,46 @@
+from leapp.actors import Actor
+from leapp.models import PCIDevices
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_with_remediation
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckPCIDevices(Actor):
```

Comment:
Is this actor supposed to be resurrected?

Reply:
nope

Reply:
Thanks for catching this...